client/service-worker.js

```diff
@@ -43,7 +43,7 @@ self.addEventListener("fetch", function (event) {
 		return;
 	}
 
-	event.respondWith(networkOrCache(event));
+	return event.respondWith(networkOrCache(event));
```

Comment:
is this actually right?

Reply:
no clue. added to try fixing HMR and forgot about it.